t/t7502-commit-porcelain.sh

```diff
@@ -262,13 +262,57 @@ test_expect_success 'cleanup commit messages (scissors option,-F,-e, scissors on
 
 	echo >>negative &&
 	cat >text <<-\EOF &&
+
 	# ------------------------ >8 ------------------------
 	to be removed
 	EOF
 	git commit --cleanup=scissors -e -F text -a --allow-empty-message &&
 	git cat-file -p HEAD >raw &&
 	sed -e "1,/^\$/d" raw >actual &&
-	test_must_be_empty actual
+	test_must_be_empty actual 
+'
+
+test_expect_success 'cleanup commit messages (scissors option,-F,-e, CR/LF line endings)' '
+
+	echo >>negative &&
+	test_config core.editor unix2dos &&
+	cat >text <<-\EOF &&
+
+	# to be kept
+
+	   # ------------------------ >8 ------------------------
+	# to be kept, too
+	# ------------------------ >8 ------------------------
+	to be removed
+	# ------------------------ >8 ------------------------
+	to be removed, too
+	EOF
```

Comment:
It might make things much more readable to define a variable containing those lengthy, repeated scissor patterns, and then use `printf` instead. For example, I have trouble seeing how line 285 differs from line 287. And I am rather certain that pretty much every reader that happens not to be you will face the exact same puzzle.

Therefore, something like this would be much preferable:

```sh
scissors="# ------------------------ >8 ------------------------" &&
test_write_lines >text \
        "# to be kept" "" "  $scissors" \
        "# to be kept, too" "$scissors." \
        "# to be removed" "$scissors" \
        "to be removed, too" &&
[...]
```

Reply:
Changes above are implemented with the whitespace is squeezed.